src/DrawerLayout.ios.js

```diff
@@ -89,6 +89,7 @@ export default class DrawerLayout extends React.Component {
       extrapolate: 'clamp',
     });
     const animatedDrawerStyles = { transform: [{ translateX: drawerTranslateX }] };
+    const drawerShown = (openValue._value > 0);
```

Comment:
Okay, I didn't thought I might do this, but it works :+1: First time with the animated API for me :rocket: